api/admin.py

```diff
@@ -1,8 +1,8 @@
 from django.contrib import admin
 
-from api.models import Categories, Journals
+from api.models import Category, Journal
 
 
 # Register your models here to view them in the admin panel.
-admin.site.register(Categories)
-admin.site.register(Journals)
+admin.site.register(Category)
+admin.site.register(Journal)
```

Comment:
later on, we plan to remove admin and auth as not necessary at the (micro)service level